CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributors
+
+## 💥 How to Contribute
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/EddieHubCommunity/LinkFree/pulls)
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/EddieHubCommunity/)
+
+- Take a look at the existing [Issues](https://github.com/EddieHubCommunity/good-first-issue-finder/issues) or [create a new issue](https://github.com/EddieHubCommunity/good-first-issue-finder/issues/new/choose)!
+- [Fork the Repo](https://github.com/EddieHubCommunity/good-first-issue-finder). Then, create a branch for any issue that you are working on. Finally, commit your work.
+- Create a **[Pull Request](https://github.com/EddieHubCommunity/good-first-issue-finder/compare)** (_PR_), which will be promptly reviewed and given suggestions for improvements by the community.
+- Add screenshots or screen captures to your Pull Request to help us understand the effects of the changes proposed in your PR.
+
+---
+
+## ⭐ HOW TO MAKE A PULL REQUEST:
+
+**1.** Start by making a Fork of the [**good-first-issue-finder**](https://github.com/EddieHubCommunity/good-first-issue-finder) repository. Click on the <a href="https://github.com/EddieHubCommunity/good-first-issue-finder/fork"><img src="https://i.imgur.com/G4z1kEe.png" height="21" width="21"></a>Fork symbol at the top right corner.
+
+**2.** Clone your new fork of the repository in the terminal/CLI on your computer with the following command:
+
+```bash
+git clone https://github.com/<your-github-username>/good-first-issue-finder
+```
+
+**3.** Navigate to the newly created good-first-issue-finder project directory:
+
+```bash
+cd good-first-issue-finder
+```
+
+**4.** Set upstream command:
+
+```bash
+git remote add upstream https://github.com/EddieHubCommunity/good-first-issue-finder.git
+```
+
+**5.** Create a new branch:
+
+```bash
+git checkout -b YourBranchName
+```
+

```

Comment:
Add a step here to explain how to add the secret github token in a .env file to get the project up and running

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

Reply:
Added the required step.